gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/SalesforceWorkUnit.java

```diff
@@ -0,0 +1,15 @@
+package org.apache.gobblin.salesforce;
+
+import java.util.Iterator;
+import java.util.List;
+import org.apache.gobblin.source.workunit.Extract;
+import org.apache.gobblin.source.workunit.WorkUnit;
+
+
+public class SalesforceWorkUnit extends WorkUnit {
```

Comment:
This class won't serialize all the state. Instead of adding this class, you should add a config key for the batch and result id list.
Something like "salesforceSource.batchAndResultList" with a value like "xxx1:zzz1,xxx2:zzz2".
Also, will you retry fetching from this state if they result is still available?

Reply:
Thanks! Hung. Will remove this class. Will also do test with `launcher.type= MAPREDUCE`. I only tested `type=LOCAL`.